App.js

```diff
@@ -0,0 +1,29 @@
+const express = require("express");
+const bodyParser = require("body-parser");
+const cors = require("cors");
+const app = express();
+const db = require("./models");
+const usersRoutes = require('./routes/users.routes')
+const postsRoutes = require('./routes/posts.routes')
+
+require('dotenv').config();
+
+// db.sequelize.sync({ force: true }).then(() => {          
+//     console.log( "Drop and re-sync db.");
+// }); 
```

Comment:
These have not been removed yet

Reply:
Actually need to use this code for recreating tables after modifications, therefore, didn't remove it, else have to search again and again.